docs/maps/api-player.md

```diff
@@ -58,6 +58,22 @@ WA.onInit().then(() => {
 })
 ```
 
+### Get the position of the player
+```
+WA.player.getPosition(): Promise<Position>
```

Comment:
The `Position` class is not documented (we should say there is a "x", and "y" properties)